.github/workflows/regression.yml

```diff
@@ -0,0 +1,30 @@
+name: Regression tests
+permissions: read-all
+
+on:
```

Comment:
can you also add the way to manually trigger them? I think I did it in the `release.yml`

Reply:
https://github.com/google/flatbuffers/blob/master/.github/workflows/release.yml#L5-L6

Reply:
nit: I wouldn't call these regression tests. Maybe just `scheduled`? or something to indicate that they are slow?

Reply:
Update, btw I also schedule this to run when PR is merged.